develop/TOOLS/DIS/Tutorials/DisValidatorContribution.md

```diff
@@ -0,0 +1,315 @@
+
+The missing test: When adding the option "datetime" to the *Measurement.Type* tag, check if the *Display* tag contains a `<Decimals>8</Decimals>` tag.
+
+Expected duration: 15 minutes.
+
+<!--
+> [!TIP]
+> See also: [Kata #X: Validator Contribution](https://community.dataminer.services/courses/kata-X) on DataMiner Dojo ![Video](~/user-guide/images/video_Duo.png)
+-->
+
+## Prerequisites
+
+- [Microsoft .NET 6.0 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/6.0)
+
+- [Microsoft Visual Studio](https://visualstudio.microsoft.com/downloads/)
+
+- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+
+- [GitHub account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account)
+
+> [!NOTE]
+> This tutorial requires [Microsoft .NET 6.0 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/6.0).
+
+> [!WARNING]
+> After installing .NET 6.0 SDK and updating Visual Studio, you may need to reboot your computer.
+
+## Step 1: Fork the repository
+
+1. Go to [https://github.com/SkylineCommunications/Skyline.DataMiner.CICD.ValidatorsExercise](https://github.com/SkylineCommunications/Skyline.DataMiner.CICD.ValidatorsExercise)
+
+1. In the top right corner, click *Fork*.
+
+1. Follow the wizard to create a fork of the repository under your own account.
+
+## Step 2: Clone your fork
+
+On the page of your GitHub fork (e.g. `https://github.com/YourGitHubHandle/Skyline.DataMiner.CICD.ValidatorsExercise`), click the *Code* button and select *Open in Visual Studio*.
+
+> [!NOTE]
+> In some cases, the *Open in Visual Studio* option may not be available. In that case, you will need to use *GitHub Desktop* instead to make the clone. Make sure you have [GitHub Desktop](https://desktop.github.com/) installed. Then click the *Code* button on your fork page and select the option *Open with GitHub Desktop* instead.
+
+## Step 3: Run the 'Validator Management Tool'
+
+1. In the *Solution Explorer*, right-click the *Validator Management Tool*, and select *Set as Startup Project*.
+
+   This will turn *Validator Management Tool* bold, and *Validator Management Tool* will appear at the top next to the *Play* button.
+
+1. Click the *Play* button.
+
+   You should now see the following screen:
+
+   ![ValidatorManagementWindow](https://github.com/SkylineCommunications/Skyline.DataMiner.CICD.ValidatorsExercise/assets/71829634/f9c6854e-205c-479c-a3fd-2e8839f46446)
+
+## Step 4: Add a new check
+
+1. In the *Validator Management Tool*, click the *Add Check...* button.
+
+   A new window will open.
+
+1. In the new window, do the following:
+
+   ![ValidatorManagementWindowCreateNewTest](https://github.com/SkylineCommunications/Skyline.DataMiner.CICD.ValidatorsExercise/assets/71829634/dcaa8bbd-6d51-43d7-8dbe-e91b3de05c73)
```

Comment:
can image be added to docs repo instead so it's self-contained?

Reply:
The same markdown is used as the readme for the exercise. The image was added there by drag & drop. Since that's a public repository, José and I tested and this should work.